BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -231,20 +262,29 @@ fileprivate final class ScrollerWrapperView: UIView {
         if self.scrollView.showsHorizontalScrollIndicator != scrollView.showsHorizontalScrollIndicator {
             self.scrollView.showsHorizontalScrollIndicator = scrollView.showsHorizontalScrollIndicator
         }
-
+        
         if self.scrollView.keyboardDismissMode != scrollView.keyboardDismissMode {
             self.scrollView.keyboardDismissMode = scrollView.keyboardDismissMode
         }
-
-        var contentInset = scrollView.contentInset
-
-        if case .refreshing = scrollView.pullToRefreshBehavior, let refreshControl = refreshControl {
-            // The refresh control lives above the content and adjusts the
-            // content inset for itself when visible. Do the same adjustment to
-            // our expected content inset.
-            contentInset.top += refreshControl.bounds.height
-        }
-
+        
+        self.applyContentInset(with: scrollView)
+    }
+    
+    private func applyContentInset(with scrollView : ScrollView)
+    {
+        // Keep a copy of the content inset as provided by the developer.
```

Comment:
What does this mean? Where are we keeping "a copy of the content inset as provided by the developer"?

Reply:
Old comment, removin